extensions/wikia/ImageReviewMercury/templates/ImageReviewMercury_index.php

```diff
@@ -0,0 +1,2 @@
+<iframe src="/image-review?useskin=wikiamobile&fullscreen=true" id="ImageReviewToolIframe"' +
```

Comment:
Sure - `&` must be converted to HTML entity in the same way `<` and `>` - refer to PHP's `htmlspecialchars()` documentation.